tests/XdebugHandlerTest.php

```diff
@@ -121,7 +122,7 @@ public function testEnvAllowWithEmptyScanDir()
         putenv('PHP_INI_SCAN_DIR');
```

Comment:
Cool, you've fixed a bug and found one! The env is already correctly set at the top of the test, so please remove this line and replace the comment above it, so:

```php
    
    // Mimic restart
    $xdebug = new XdebugHandlerMock($loaded);
```

Reply:
Now everything is working. Sorry for the mess :sweat_smile:

Reply:
Many thanks for sorting it.